test/dynamo/test_dynamic_shapes.py

```diff
@@ -83,8 +92,13 @@ def make_dynamic_cls(cls, *, static_default=False):
 for test in tests:
     make_dynamic_cls(test)
     make_dynamic_cls(test, static_default=True)
+    make_dynamic_cls(
+        test,
+        static_default=True,
+        automatic=config.DYNAMIC_SHAPE_STRATEGY.ALL_FAILED_IN_FRAME,
+    )
```

Comment:
I think my pref is to fold this into static_default, so we don't go from 2x to 3x tests. WDYT.

Reply:
I do not have a strong opinion. The reasoning here was, the tests are fast, and there are cases when you use one, but not the other, like export.

Reply:
But it export you only run the model once anyway, so it doesn't matter anyway. (Also, in export, you probably don't want to consult the frame state anyway! Which I don't see code for doing yet.)